src/applications/disability-benefits/all-claims/pages/alternateNames.js

```diff
@@ -0,0 +1,66 @@
+import FullNameField from 'us-forms-system/lib/js/fields/FullNameField';
+
+export const uiSchema = {
+  'view:hasAlternateName': {
+    'ui:title': 'Did you serve under another name?',
+    'ui:widget': 'yesNo'
+  },
+  alternateNames: {
+    'ui:description': 'What names did you serve under?',
+    'ui:options': {
+      viewField: FullNameField,
+      reviewTitle: 'Other names',
+      expandUnder: 'view:hasAlternateName'
+    },
+    items: {
+      first: {
+        'ui:title': 'First name',
+      },
+      middle: {
+        'ui:title': 'Middle name',
+      },
+      last: {
+        'ui:title': 'Last name',
+      }
+    }
+  }
+};
+
+export const schema = {
+  type: 'object',
+  required: ['view:hasAlternateName'],
+  properties: {
+    'view:hasAlternateName': {
+      type: 'boolean'
+    },
+    alternateNames: {
+      type: 'array',
+      minItems: 1,
+      maxItems: 100,
+      items: {
+        type: 'object',
+        required: ['first', 'last'],
+        properties: {
+          first: {
+            type: 'string',
+            minLength: 1,
+            maxLength: 30,
+            pattern: "^([a-zA-Z0-9-/']+( ?))+$"
```

Comment:
pattern & length requirements taken from EVSS `submit` swagger

Reply:
I think we would save ourselves a step later if we still wrote the pages as the schema was in a separate file in vets-json-schema, and just kept that file in vets-website. That's the approach that the initial version of the pre-need form took. It saves you from having to delete all of this code and replace it with the imported schema.

Reply:
Yeah, that seems like a good call.